packages/css/src/paragraph/README.md

```diff
@@ -0,0 +1,36 @@
+# Paragraph
+
+De paragraph component gebruik je voor een alinea met lopende tekst, instructies van formulieren en voor overige losstaande tekstfragmenten.
+
+## Richtlijnen
+
+- Groepeer zinnen die bij elkaar horen in een alinea. Begin een nieuwe alinea wanneer de tekst over een andere onderwerp gaat of een eigen doel heeft.
+- Overweeg of een alinea met meer dan 7 zinnen of 140 woorden duidelijker is als je tekst indeelt in twee alinea's. Teksten met niet al te lange alinea's zijn makkelijker te begrijpen als je pauze neemt, en door het groeperen is informatie sneller terug te vinden.
+
+## Groottes (sizes)
+
+### Large
```

Comment:
We might want input from UX for this as well? The less technical part of what and how?

Reply:
Yeah, definitely, I think UX should (also) do the review for these documentation PRs